02-parsing/ospf2.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Library   Collections  # Imports a public Robot framework library called 'Collections'
+Library   CXTA
+Resource  cxta.robot
+
+# the following causes a specific keyword to execute before any of the tests
+# are run
+Suite Setup     suite-setup
+
+*** Test Cases ***
+
+Get the ospf neighbour ID from R1 using TextFSM keywords
+    select device "r1"
+    # runs a command and parses it through the TextFSM template provided
+    # cxta includes a large collection of templates (view the 'Command Map' page in the documentation
+    # though you are able to specify your own template, as we do in this example
+    run parsed "show ip ospf neighbor" with template "02-parsing/show_ip_ospf_neighbor.textfsm"
```

Comment:
Not sure if @oboehmer  would like a different path here or using currentdir or anything.